macros/extract_url_parameter.sql

```diff
@@ -0,0 +1,20 @@
+{% macro extract_url_parameter(field, url_parameter) -%}
+
+{{ adapter.dispatch('extract_url_parameter', 'fivetran_utils') (field, url_parameter) }}
+
+{% endmacro %}
+
+
+{% macro default__extract_url_parameter(field, url_parameter) -%}
+
+{{ dbt_utils.get_url_parameter(field, url_parameter) }}
+
+{%- endmacro %}
+
+
+{% macro databricks__extract_url_parameter(field, url_parameter) -%}
+
+{%- set formatted_url_parameter = "'" + url_parameter + "=([^&]+)'" -%}
+nullif(regexp_extract({{ field }}, {{ formatted_url_parameter }}, 1), '')
```

Comment:
So it looks like the only change is that you removed both `split()` macros that were initially found in the compiled version of the databricks macro. What was the split being used for before? Are we possibly not taking something into consideration with why the split was there in the first place?

Reply:
Summarizing our discussion from earlier:
- The regexp_extract() in the prior version was used to check for special characters, while the split() did the work of extracting the url parameter
- In this version, the regexp_extract() replaces the split() to do the work of extracting the url parameter, and it also automatically handles special characters.  
- I tested in our and the customer's environments, and it works for both our cases

Reply:
Thanks for the explanation. This sounds good to me!